format.h

```diff
@@ -2773,7 +2789,7 @@ void BasicWriter<Char>::write_double(
           spec.width() > static_cast<unsigned>(n)) {
         width = spec.width();
         CharPtr p = grow_buffer(width);
-        std::memmove(get(p) + (width - n) / 2, get(p), n * sizeof(Char));
+        std::uninitialized_copy(p, p + n, p + (width - n) / 2);
```

Comment:
This overwrites the latest changes. Please rebase your branch to the current master.